src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -367,6 +378,105 @@ public static double getBlockHeight(final int type, final byte data) {
 				return 1;
 		}
 	}
+

```

Comment:
Maybe put these utility methods in a different class?

Reply:
a new one or do you have a suggested place?